common/src/main/resources/assets/gtceu/compass/pages/en_us/circuits/neutronium_wafer.xml

```diff
@@ -1,152 +1,36 @@
-  <br/>
-  •
-  <style>
-   can be combined with a pink lens to create
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">NOR wafer</style>
-   Nitrosyl Chloride - Nitrosyl Chloride
-  </style>
-  <br/>
-  <br/>
-  •
-  <style>
-   can be combined with a brown lens to create
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">PIC chip</style>
-   ;
-  </style>
-  <br/>
-  <br/>
-  •
-  <style>
-   can be combined with a yellow lens to create
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">SoC chip</style>
-   ;
-  </style>
-  <br/>
-  <br/>
-  •
-  <style>
-   Wood Tar - Wood Tar
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">
- Biomass - Biomass
-</style>
-   Titanium Carbide - TitaniumCarbide - TiC
-  </style>
-  <br/>
- </text>
- <text>
-  <br/>
-  <style>
-   Among the above-mentioned precision laser etching machine formulas, the formula power inherited from silicon wafers totals 384,000 EU, inherited from
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">Fluorite-doped wafers</style>
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">
- Magnetized neodymium - NeodymiumMagnetic
-</style>
-   The formula power of the silica doped wafer totals 3,840,000 EU, the remaining formula power totals 6,912,000 EU, and the energy consumption power is 7,680 EU/t (IV), inherited from
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">silicon wafer</style>
-   ,
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">
- can also be mixed in it; 
-</style>
-   Chromite - Chromite
-   <style color="0xafff00" hover-info="click link" link="gtceu:links" underlined="true">Silica doped wafers</style>
-   The recipe time for silica-doped wafers is reduced to 25s, and the rest takes 45s. No lenses are consumed and a clean room is required.
-  </style>
-  <br/>
- </text>
+  <h1>
+    Neutronium-doped Wafer
+  </h1>
+  <h3>
+   Source
+  </h3>
+  <text><style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br/>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/179043/">花海</style>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://github.com/Yefancy">KilaBash</style><br/></style></text>
+  <text>
+    Neutronium-doped Wafer, an advanced wafer material.
+  </text> 
+  <br/>
+  <h3>Source</h3>
+  <text>
+    Neutronium-doped Wafers are produced by cutting <style color="0xafff00" underlined="true" link="gtceu:circuits/neutronium_boule" hover-info="click link">Neutronium-doped monocrystalline Silicon Boule</style> in a <style color="0xafff00" underlined="true" link="gtceu:machines/cutter" hover-info="click link">cutter</style> at the lowest IV-Tier voltage. 
```

Comment:
should say `Neutronium-doped Wafers are produced by cutting a Neutronium-doped monocrystalline Silicon Boule in an at least IV-tier cutter` (with all the formatting OFC)

Reply:
same for the other wafer pages